app/javascript/lib/commons/index.js

```diff
@@ -1,53 +1,37 @@
-/**
- *
- * POLYFILLS
- *
- */
+// POLYFILLS
 import "core-js/stable";
 import "regenerator-runtime/runtime";
 
+// https://rossta.net/blog/importing-images-with-webpacker.html#images-in-rails-views
+require.context('../../images', true)
+
 // https://attacomsian.com/blog/javascript-dom-detect-internet-explorer-browser
 if (document.documentMode) {
   import("css-vars-ponyfill").then(({ default: cssVars }) => {
     cssVars() // Allow IE use CSS custom variables. Initialization
   })
 }
 
-/**
- *
- * DEPENDENCIES
- *
- */
+// LEGACY DEPENDENCIES
 import $ from 'jquery'
 import 'jquery-ujs'
 import * as I18n from 'i18n-js'
-import Turbolinks from 'turbolinks'
-import { handleIFramePageLoaded } from "../shared/modules/iframe_handler.js";
 
-//NOTE: these modules are required
-import "./modules/module-search.js";
-import "./modules/module-sessions.js";
-import "./modules/module-site_header.js";
-import "./modules/magnific-popup.js";
-import "./modules/tabs.js";
-import "./modules/velocity_settings.js";
-import "./modules/shareContent.js";
-import "./modules/separate-tabs.js";
+// NOTE: these modules are required for the legacy views
+import "./modules/module-search";
+import "./modules/module-sessions";
+import "./modules/module-site_header";
+import "./modules/magnific-popup";
+import "./modules/tabs";
+import "./modules/velocity_settings";
+import "./modules/shareContent";
+import "./modules/separate-tabs";
+import "./modules/disable-turbolinks";
```

Comment:
I think a better name would be turbolinks, because you don't always disable turbolinks!

Reply:
`disable-turbolinks-if.js` :trollface:

Reply:
Changed